recipe/conda_build_config.yaml

```diff
@@ -10,3 +10,5 @@ cxx_compiler:                  # [win]
   - vs2017                     # [win]
 vc:                            # [win]
   - 14.1                       # [win]
+channel_targets:               # [not (osx and arm64)]
+  - conda-forge python_rc      # [not (osx and arm64)]
```

Comment:
Why not for `osx and arm64`?

Reply:
`osx and arm64` is an experimental platform and therefore release candidates are okay there. Need this on arm64.

Reply:
okay